core/src/test/groovy/com/shazam/tocker/ExposesEnvironmentVariablesSpec.groovy

```diff
@@ -0,0 +1,47 @@
+package com.shazam.tocker
+
+import com.shazam.tocker.dsl.DockerDsl
+import spock.lang.Specification
+
+
+class ExposesEnvironmentVariablesSpec extends Specification implements DockerDsl {
+    
```

Comment:
These tests names differ from the standard elsewhere in the project.  Could they be changed please?

Reply:
Changed - is that better?

Reply:
Awesome, thanks!